packages/StepTracker/StepTracker.jsx

```diff
@@ -1,95 +1,70 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import classNames from 'classnames'
+
+import Text from '@tds/core-text'
+import Flexbox from '../../shared/components/Flexbox/Flexbox'
 
 import Step from './Step/Step'
 
-import './StepTracker.scss'
+import styles from './StepTracker.modules.scss'
 
 /**
  * Show the current position in a sequence of steps.
  *
  * @version ./package.json
  */
-class StepTracker extends Component {
-  getStatus(current, index) {
-    if (index < current) {
-      return 'completed'
-    } else if (index === current) {
-      return 'processing'
-    }
 
-    return 'waiting'
+const StepTracker = ({ current, steps, mobileStepLabelTemplate }) => {
+  const parseStepText = () => {
+    return mobileStepLabelTemplate
+      .replace('%{current}', current < steps.length ? current + 1 : steps.length)
+      .replace('%{total}', steps.length)
   }
 
-  render() {
-    const { children, className, current } = this.props
-    const cls = classNames('step-tracker', className)
-    const totalSteps = children.length
-
-    let currentStepLabel
-    let currentStepNumber
-
-    if (current < totalSteps) {
-      currentStepLabel = children[current].props.label
-      currentStepNumber = current + 1
-    } else {
-      currentStepLabel = children[current - 1].props.label
-      currentStepNumber = current
-    }
-
-    return (
-      <div role="progressbar">
-        <ul className={cls}>
-          {React.Children.map(
-            children,
-            (element, index) => {
-              const stepNumber = index + 1
-              const status = this.getStatus(current, index)
-              const props = {
-                stepNumber,
-                status,
-              }
-              return React.cloneElement(element, props)
-            },
-            this
-          )}
-        </ul>
-        <div className="step-tracker__mobile-label">
-          <span className="step-tracker__mobile-label-step-info">
-            Step {currentStepNumber} of {totalSteps}:
-          </span>{' '}
-          {currentStepLabel}
-        </div>
-      </div>
-    )
+  const getStepLabel = () => {
```

Comment:
Let's hoist these functions outside of the component definition. The convention is to put all functions at the same level in the file (aka module). This also makes it more clear that these functions have specific input and output.